packages/elements-core/src/hooks/useResizer.ts

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+export function useResizer(sidebarWidth: number): [React.Ref<HTMLDivElement>, number, (event: any) => void] {
```

Comment:
Seems this hook consider only to `SidebarLayout` component and should be private